src/rejection_intake.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+if (!defined('not_direct_access') && !empty($_POST['pre']) && !empty($_POST['post']) && !empty($_REQUEST['hunter_id_hash'])) {
+    define('not_direct_access', TRUE);
+    require_once "check-cors.php";
+    require_once "uuid.php";
+    require_once "check-ban.php";
+    require_once "check-time.php";
+    require_once "config.php";
+    require_once "db-connect.php";
+    require_once "send_response.php";
+    recordRejectionsInFile($_POST['pre'], $_POST['post']);
+    sendResponse('success', "Thanks for the hunt info!");
+}
+
+sendResponse('success', "Thanks for the hunt info!");
```

Comment:
you are sending response twice if that if above executes. So maybe we should post this into an "else" with an error log, to make sure we catch any messages that come and fail to meet the criteria.

Reply:
and die() so it doesn't continue execution.